autograder/util/load.py

```diff
@@ -1,25 +1,37 @@
-# Read a TSV file and return a list of lists of the cleaned fields in the TSV file.
-# Raise an error if a line has more fields than the maximum length.
-def load_users(path, max_len):
-    all_parts = []
+# Read a TSV file and return a list of lists of the stripped fields in the TSV file.
+# Raise an error if a line has more fields than the maximum length
+# or less fields than the minimum length.
+def load_tsv(path, max_len, min_len = 1):
+    rows = []
 
     with open(path, 'r') as file:
         lineno = 0
         for line in file:
             lineno += 1
 
             line = line.strip()
+
             if (line == ""):
-                continue
+                if (min_len > 0):
```

Comment:
Empty lines should just be ignored.
We only need to check min_len if there are things to check.